### PR TITLE
Added location to hangouts, fixed chapter-member routes, docs update

### PR DESCRIPTION
**Pull request by markd315**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Added location to hangouts, fixed chapter-member routes, docs update

* **What is the current behavior?** (You can also link to an open issue here)
No location or requirement for hangouts, malformed chapter member route with duplicate data, fewer docs

### CODE REVIEW

File: test/unit/hangout/hangout.validator.spec.ts [hunk 1]
```diff
@@ -85,6 +114,7 @@ describe('Hangout Validator Tests', () => {
   it('Should pass for a good date', () => {
     hangout.title = "title"
     hangout.date = "2999-11-20";
+    hangout.location = "Phase 10";
```
Comment (markd315) [inline test/unit/hangout/hangout.validator.spec.ts]: This is pretty weird, but if I don't put this line in a couple of tests go red, I don't think the test object is getting fully reset between executions.

Something to look into maybe, but resetting fields manually when it happens seems to work 100% of the time.

Comment (alindenberg) [reply test/unit/hangout/hangout.validator.spec.ts]: You need to move that `hangout = Object.assign({}, defaultHangout)` out of the `before()` block and into the `beforeEach()` block to have it reset after each test case

Comment (alindenberg) [reply test/unit/hangout/hangout.validator.spec.ts]: That may have been me that put it there initially (I'm honestly unsure) but it should be in the `beforeEach()` either way. Possibly the `search` object too(?) but I'm not sure of its use-case so can't speak to it

Comment (markd315) [reply test/unit/hangout/hangout.validator.spec.ts]: You're right, I'll get it real quick

**Pull request merged**